root/inc/river-gauge.html

```diff
@@ -2,9 +2,9 @@
   Unlike an <img>, an <object> grafts the SVG document into the DOM, which
   means browsers will display the <title> elements of the SVG.  Yay!
 -->
-<object data="/river/gauge/<% distribution | uri | html %>"
+<object data="/river/gauge/<% distribution | uri %>"
```

Comment:
Relying on `uri` to do enough escaping for HTML too is bad practice.  Why did you remove it here?

Reply:
I was checking on how the default filtering is applied and it slipped into the commit.